test/Cardano/CoinSelection/LargestFirstSpec.hs

```diff
@@ -229,8 +233,10 @@ propDeterministic
     -> Property
 propDeterministic (CoinSelProp utxo txOuts) = do
     let opts = CoinSelectionOptions (const 100) noValidation
-    let resultOne = runIdentity $ runExceptT $ largestFirst opts txOuts utxo
-    let resultTwo = runIdentity $ runExceptT $ largestFirst opts txOuts utxo
+    let resultOne = runIdentity $ runExceptT $
+            selectCoins largestFirst opts txOuts utxo
+    let resultTwo = runIdentity $ runExceptT $
+            selectCoins largestFirst opts txOuts utxo
     resultOne === resultTwo
```

Comment:
I suspect that this property is actually redundant, since `largestFirst` is a pure function.

Reply:
This property has since been modified to be useful again.